apps/studio/electron/preload/webview/dom.ts

```diff
@@ -42,10 +42,17 @@ export function buildLayerTree(root: HTMLElement): Map<string, LayerNode> | null
 
     const layerMap = new Map<string, LayerNode>();
     const treeWalker = document.createTreeWalker(root, NodeFilter.SHOW_ELEMENT, {
-        acceptNode: (node: Node) =>
-            isValidHtmlElement(node as HTMLElement)
```

Comment:
You could just update this to check for the parent being svg and node being path
https://github.com/onlook-dev/onlook/blob/2e40d6a2c8c69ca6582b2aff4459ea569ea950a3/packages/models/src/constants/index.ts#L159